Payments pallet blog post (#13)
virto-network/website: blog/01_payments-pallet-grant.md

>*Web3 Foundation funds research and development teams building the technology
>stack of the decentralized web. It was established in Zug, Switzerland by
>Ethereum co-founder and former CTO Gavin Wood. Polkadot is the Foundation's
>flagship project.*

**Virto Network** aims to build the best decentralized infrastructure for
marketplaces with local impact in the Polkadot ecosystem. To ensure the success
of marketplaces, it's Virto's responsibility and pursuit to provide
marketplaces with secure and convenient payment system. With this goal and
expectation **Virto Payment Pallet** was born. 

**Virto Payment Pallet** tries to tell people a story of trust, which starts
with **trustless**. Imagine you want to start your own business, what would be
the most challenging part? Many answers may already come to mind, among which
the word **trust** appears often. For example, how could you build the trust
with your business partners? How could you gain trust from your customers every
time they purchase from you? How could your trust other services and goods
whenever you buy? We know it's not easy to start with just naive trust, that's

> why Virto purpose to build the Payment Pallet with the trustless design. People

Reviewer: ```suggestion
why Virto's purpose is to build the Payment Pallet with a trustless design. People
```

could pay each other easily using the pallet. The payment from buyers will
always be locked first and only be released to sellers once the buyers
confirmed to have received the product or services. The payment pallet also
allows the creator of the payment to trigger cancel with a buffer time and the
recipient to dispute the payment request of sender.   

> How does Virto Payments Pallet achieve the desired functionality? As the

Reviewer: You use different terms for the same concept:

-  Payment Pallet
- Payments Pallet
- payments pallet

It'd be more consistent to use the same term everywhere, whenever the same concept is describe.

Reviewer 2: Thanks, yeah the correct way would be "Payments Pallet", we'll look into that

fundamental part of [LIP][lip], it's an escrow like system that allows people

> pay with cryptocurrency for any product or service from the real world, it has

Reviewer: ```suggestion
to pay with cryptocurrency for any product or service from the real world, it has
```

been redesigned from the ground up to be highly configurable to serve the needs
of the Virto Network as well as any other Substrate based chain that wants to
engage with real-world assets.

We are very excited about what this milestone represents for the future of
Virto Network, payments pallet is just the first step in our path towards

> making blockchain technology usable in the day to day live of common users and

Reviewer: ```suggestion
making blockchain technology usable in the day to day life of common users and
```

making _decentralized commerce with social impact_ the default way of conducting

> buissiness in the world of Web3. 

Reviewer: ```suggestion
business in the world of Web3. 
```
